doc/data/messages/c/consider-using-f-string/bad.py

```diff
@@ -0,0 +1,5 @@
+menu = ('eggs', 'spam')
+
+order = "%s and %s" % menu # [consider-using-f-string]
+
+new_order = "{1} and {0}".format(menu[0], menu[1]) # [consider-using-f-string]
```

Comment:
```suggestion
old_order = "%s and %s: %.2f ¤" % menu # [consider-using-f-string]
beginner_order = menu[0] + "and " + menu[1] + ": " + str(menu[2]) + " ¤"
joined_order = "and ".join(menu) 
format_order = "{} and {}: {:0.2f} ¤".format(menu[0], menu[1], menu[2]) # [consider-using-f-string]
named_format_order = "{eggs} and {spam}: {price:0.2f} ¤".format(eggs=menu[0], spam=menu[1], price=menu[2]) # [consider-using-f-string]
template_order = Template('$eggs and $spam: $price ¤').substitute(eggs=menu[0], spam=menu[1], price=menu[2])
```

Reply:
Hmm, thanks for introducing me to the `¤` sign!